styles/index.css

```diff
@@ -0,0 +1,15 @@
+@import url(/styles/utility.css);
+@import url(https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css);
+
+.img-size {
```

Comment:
name your classes in a more meaningful way so that when you come back to your code it will be easy to refactor and catch bugs

Reply:
Thanks for the comment, changed the class name